src/index.js

```diff
@@ -93,10 +97,6 @@ const App = () => {
     await api.minusOne({ lineItem, lineItems, setLineItems, cart });
   };
 
-  const decrement = async (lineItem) => {
```

Comment:
is the decrement supposed to be just deleted

Reply:
Yeah. It was a temp name for something originally